Invoke-CommandAs/Private/Invoke-ScheduledTask.ps1

```diff
@@ -29,6 +29,27 @@ function Invoke-ScheduledTask {
                 $JobParameters['ScheduledJobOption'] = New-ScheduledJobOption -RunElevated
             }
 
+            # Can't create a schedule job if you're running as Local System.
+            # Workaround to create a temporary user to create the schedule job.
+            # Cleanup the temporary user after the job is done.
+            if ("NT AUTHORITY\SYSTEM" -eq [System.Security.Principal.WindowsIdentity]::GetCurrent().Name) {
+                $schedulerPass = ConvertTo-SecureString (New-Guid).Guid -AsPlainText -Force
+                $schedulerName = "service.scheduler"
+    
+                # Check if the temporary user already exists.
+                # If it doesn't exist then create it.
+                # Else we have to update the password so we can create a credential.
+                if ($false -eq [bool](Get-WmiObject Win32_UserAccount -Filter "LocalAccount='true' and Name='$schedulerName'")) {
```

Comment:
Not sure if this PR is dead but I wanted to point out:

This will most likely fail on PSCore since it uses `Get-CimInstance` instead of `GetWmiObject`